packages/sync-server/src/sync/CentralSyncManager.ts

```diff
@@ -244,12 +247,12 @@ export class CentralSyncManager {
 
       // will wait for concurrent snapshots to complete if we are currently at capacity, then
       // set the snapshot_started_at timestamp before we proceed with the heavy work below
-      await startSnapshotWhenCapacityAvailable(this.database, sessionId);
+      await startSnapshotWhenCapacityAvailable(this.models.database, sessionId);
 
       // get a sync tick that we can safely consider the snapshot to be up to (because we use the
       // "tick" of the tick-tock, so we know any more changes on the server, even while the snapshot
       // process is ongoing, will have a later updated_at_sync_tick, i.e. the "tock")
-      const { tick } = await this.tickTockGlobalClock();
+      const { tick } = await this.tickTockGlobalClock(this.models);
```

Comment:
Oh right, ignore previous comment - at least some of the time, we aren't using transacting models